code/modules/fallout/reagents/medicines.dm

```diff
@@ -53,9 +53,6 @@
 		M.AdjustKnockdown(-5*REAGENTS_EFFECT_MULTIPLIER, 0)
 		M.adjustStaminaLoss(-2*REAGENTS_EFFECT_MULTIPLIER)
 		. = TRUE
-	if(M.nutrition <= NUTRITION_LEVEL_STARVING - 50)
```

Comment:
But why?

Reply:
It didn't really make much sense for stimpaks to kill you if you were hungry and for supr stims to drain your nutrition, considering powder poultice and bitter drink...didn't.